java/build-android.gradle

```diff
@@ -137,8 +149,9 @@ publishing {
 			artifact sourcesJar
 
 			pom {
-				name = enableTrainingApis ? 'onnxruntime-training' : 'onnx-runtime'
-				description = enableTrainingApis ? trainingDescription : defaultDescription
+				name = qnnVersion != null ? 'onnxruntime-qnn' : (enableTrainingApis ? 'onnxruntime-training' : 'onnx-runtime')
+				description = qnnVersion != null ? QNNDescription : (enableTrainingApis ? trainingDescription : defaultDescription)
```

Comment:
nit: as we have 'qnnVersion' would 'qnnDescription' be more consistent naming with everything else?